web/src/components/tasks/business-formation/BusinessFormationPaginator.tsx

```diff
@@ -458,7 +459,35 @@ export const BusinessFormationPaginator = (props: Props): ReactElement => {
         );
       });
 
-      return (
+      const fieldsWithErrors = dedupedFieldErrors.map((fieldError) => {
+        let configFieldName = fieldError.field as ConfigFormationFields;
+
+        if (fieldError.field === "members") {
+          configFieldName = getConfigFieldByLegalStructure(state.formationFormData.legalType);
+        }
+
+        // eslint-disable-next-line @typescript-eslint/no-explicit-any
+        const label = (Config.formation.fields as any)[configFieldName].label;
+        return {
+          name: configFieldName,
+          label,
+          children: getApiErrorMessage(fieldError.field) && (
+            <ul>
+              <li>{getApiErrorMessage(fieldError.field)}</li>
+            </ul>
+          ),
+        };
+      });
+
+      const swap = true;
```

Comment:
[dust] This shouldn't be hard coded

Reply:
That's actually a [boulder]. I had added that to quickly switch between the 2 options in the ternary below.